### DISCUSSION TITLE
Proposal: Getting JSON Schema into Hacktoberfest'24 🚀

### DISCUSSION THREAD

**Discussion by DhairyaMajmudar** [json-schema-org/community#801] [Vision and direction]
# Hacktoberfest 2024

[Hacktoberfest](https://hacktoberfest.com) is an open-source event that takes place every year in the month of October. Several open source developers worldwide take part in the event and contribute the code for different projects.

It will be a great learning opportunity for new/experienced developers to contribute code in JSON Schema projects which will also further help us (JSON Schema) to improve our projects while taking help from the open-source community. 

## Procedure to take part in Hacktoberfest as maintainer

1. Make an announcement of JSON Schema projects taking part in the [discord](https://discord.com/invite/hacktoberfest) channels of Hacktoberfest.
2. Finding/Creating issues that can be worked upon in one month of timeline.
3. Labelling those issues and PR resolving them with `hacktoberfest` and `hacktoberfest-accepted` respectively.

Reference [Hacktoberfest website](https://hacktoberfest.com/participation/#maintainers)

**Comment by benjagm** [comment]
Sounds good to me. Can you drive this Dhairya?  If that is the case let's do it.

**Reply by benjagm** [reply]
We need to add another action to promote this within the community

**Reply by DhairyaMajmudar** [reply]
Thank you, Yes I would like to drive this : )

**Reply by heysujal** [reply]
It will be great!

**Reply by JeelRajodiya** [reply]
Hello, the [tour](https://github.com/json-schema-org/tour) repo is also participating in the event.
